shared/common-adapters/dropdown.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import * as React from 'react'
+import Box from './box'
+import PopupDialog from './popup-dialog'
+import Icon from './icon'
+import {globalStyles, globalColors, globalMargins, glamorous} from '../styles'
+import {isMobile} from '../constants/platform'
+
+type Props = {
+  onChanged: (selected: React.Node) => void,
+  selected?: React.Node,
+  items: Array<React.Node>,
+  style?: any,
```

Comment:
`Object` instead of `any`?

Reply:
its actually more complex than an object, it can be an array etc. no point in really strongly typing this anyways in practice